src/main/java/com/google/sps/servlets/getUserImagesForQuizPage.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.servlets;
+ 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.sps.utils.QuizTimingPropertiesUtils;
+import com.google.sps.utils.UserUtils;
+import java.io.*;
+import javax.servlet.*;
+import javax.servlet.http.*;
+import java.util.ArrayList;
+import java.util.HashMap;
+ 
+//This servlet loads user images to the quiz page for a logged-in user 
```

Comment:
Which images?
Please add something like (my interpretation): "This returns a JSON map from user ID to image blob key for each player in the current game, excluding the logged-in player."

Also, why exclude the logged-in player?

Reply:
we don't want the user to be able to vote for their own picture in the quiz, I added documentation to help explain this...let me know if it helps

Reply:
What's the chain of logic between "we don't want the logged-in user to vote for their own game" and "therefore, we don't return the user's own picture here"?

I'm worried you're selecting a pair of pictures to judge in the JavaScript code. That would be a security bug

Reply:
The idea is that the pictures of each user playing that game will be loaded onto the quiz page except for the one that belongs to the current user

Reply:
Oh okay. I thought it was a matter of selecting a pair, but now I understand you display every picture all at once.